locust/configuration.py

```diff
@@ -58,20 +59,20 @@ def read_json(self):
                 self.config_data = json.load({})
         return self.config_data
 
-    def update_json_config(self, json_added, json_path, options, list_column, config_text):
+    def update_json_config(self, data_json, json_added, json_path, options, list_column, config_text):
         """
         Write JSON file configuration
         """
-        data = literal_eval(config_text)
+        data = data_json
 
         if(options != "replace"):
             json_target = jsonpath_rw_ext.match(json_path, data)
```

Comment:
for example this one. rather than use another variable (`data`) which has the same value as data_json, it is better to use the data_json